.stickler.yml

```diff
@@ -0,0 +1,10 @@
+linters:
+  rubocop:
+    display_cop_names: true
+files:
+  ignore:
+    - "bin/*"
+    - "db/*"
+    - "config/*"
+    - "Guardfile"
+    - "Rakefile"
```

Comment:
Let's make sure **Stickler** is checking this repo:
- Install **Stickler** on this repo
- `On` this repo on [**Stickler** website](https://stickler-ci.com/)

  ![on stickler](https://user-images.githubusercontent.com/36057474/65038555-17b32580-d948-11e9-8fe0-a2b3b1e5a863.gif)